lego/apps/events/serializers/pools.py

```diff
@@ -28,9 +28,14 @@ def create(self, validated_data):
 
     def get_registrations(self, obj):
         queryset = obj.registrations.all()
-        if obj.event.is_priced:
-            return RegistrationPaymentReadSerializer(queryset, context=self.context, many=True).data
-        return RegistrationReadSerializer(queryset, context=self.context, many=True).data
+        should_see_regs = self.context.get('should_see_regs', None)
+        if should_see_regs:
+            if obj.event.is_priced:
+                return RegistrationPaymentReadSerializer(
+                    queryset, context=self.context, many=True
+                ).data
+            return RegistrationReadSerializer(queryset, context=self.context, many=True).data
+        return -1
```

Comment:
This can be changed to return something more meaningful, such as number of attendees, but I think this is enough for now